packages/deployment/iam/cloudformation.template.yml

```diff
@@ -652,6 +652,7 @@ Resources:
                   - ecs:UpdateContainerInstancesState
                   - lambda:GetFunction
                   - lambda:invokeFunction
+                  - lambda:GetLayerVersion
```

Comment:
This is needed to allow the cumulus-ecs-task task to pull down layer version code